ProjectManager/ProjectManager/Common/View/TableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TableViewCell.swift
+//  ProjectManager
+//
+//  Created by seohyeon park on 2022/09/12.
+//
+
+import UIKit
+
+class TableViewCell: UITableViewCell {
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var deadlineLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+
+    func setUpCell(data: TaskModel) {
+        titleLabel.text = data.taskTitle
+        descriptionLabel.text = data.taskDescription
+        deadlineLabel.text = data.taskDeadline
+
+        if checkPastDate(date: data.taskDeadline) {
+            deadlineLabel.textColor = .red
```

Comment:
checkPastDate 메소드를 TaskModel  로 빼서 아래처럼 만들어보는건 어떨까요?

```suggestion
            deadlineLabel.textColor = data.deadLineLabelTextColor
```

Reply:
제안해주신 코드로 수정했습니다!